contrib/src/test/java/com/datatorrent/contrib/hbase/HBaseTransactionalPutOperatorTest.java

```diff
@@ -55,44 +59,11 @@ public void testAtleastOnce() throws Exception {
       t1.setColFamily("colfam0");t1.setColName("street");t1.setRow("row1");t1.setColValue("ts");
       HBaseTuple t2=new HBaseTuple();
       t2.setColFamily("colfam0");t2.setColName("city");t2.setRow("row2");t2.setColValue("tc");
-      thop.setup(new OperatorContext() {
-
-        @Override
-        public <T> T getValue(Attribute<T> key) {
-          if(key.equals(PROCESSING_MODE)){
-            return (T) ProcessingMode.AT_LEAST_ONCE;
-          }
-          return key.defaultValue;
-        }
-
-        @Override
-        public AttributeMap getAttributes() {
-          return null;
-        }
-
-        @Override
-        public int getId() {
-          // TODO Auto-generated method stub
-          return 0;
-        }
-
-        @Override
-        public void setCounters(Object counters) {
-          // TODO Auto-generated method stub
-
-        }
-
-        @Override
-        public void sendMetrics(Collection<String> collection)
-        {
-        }
-
-        @Override
-        public int getWindowsFromCheckpoint()
-        {
-          return 0;
-        }
-      });
+      AttributeMap.DefaultAttributeMap attributeMap = new AttributeMap.DefaultAttributeMap();
+      attributeMap.put(OperatorContext.PROCESSING_MODE, ProcessingMode.AT_LEAST_ONCE);
+      attributeMap.put(OperatorContext.ACTIVATION_WINDOW_ID, -1L);
```

Comment:
This was as @tweise had suggested to use mockOperatorContext. Should we move it to different PR?
We will still have to make some changes in this file since there are compilation issues.